app/uk/gov/hmrc/securemessage/controllers/models/generic/ConversationMetadata.scala

```diff
@@ -72,18 +68,15 @@ object ConversationMetaData {
 
   private def findUnreadMessagesByParticipant(participantId: Int, coreConversation: Conversation): Boolean = {
     val messages = coreConversation.messages
-    val matchingIds = coreConversation.messages
-      .flatMap(_.readBy)
-      .map(_.id)
-      .count(int => int === participantId)
-    matchingIds =!= messages.size
+    val messagesReadByParticipant = messages.filter(_.readBy.map(_.id).contains(participantId))
```

Comment:
Name of function is find but it returns a  boolean

Reply:
I think that's a merge issue, I thought @matthewgroom fixed that name?

Reply:
That's my fault Murali. I thought I changed it to anyUnreadMessages. Sorry @rwalpole

Reply:
Updated.